drivers/sub8_driver_utils/camera_utils/CMakeLists.txt

```diff
@@ -1,20 +1,25 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(camera_utils)
 
+# This builds a utility to reset the camera bus using the DC1394 camera libraries
+# Since we are moving away from dc1394 towards pointgrey drivers. This utility is 
+# no longer applicable. HoWever, we may need to create something similar using the
+# flycapture SDK
```

Comment:
Hopefully not, I haven't had any errors that restarting the camera nodelet didn't fix.